src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -1534,6 +1560,7 @@ private bool CollectArgs(string[] args, ref int i)
         private bool _noPrompt;
         private string? _commandLineCommand;
         private bool _wasCommandEncoded;
+        private bool _commandHasArgs = false;
```

Comment:
We have no need to init to default.
```suggestion
        private bool _commandHasArgs;
```